lib/tasks/argo.rake

```diff
@@ -18,6 +18,7 @@ end
 task :default => [:ci]
 
 task :ci do
+  raise 'NEVER RUN CI ON PROD' if ENV['RAILS_ENV'] =~ /production/i
   ENV['RAILS_ENV'] = 'test'
```

Comment:
I prefer to be paranoid and leave the one-liner as is, regardless of the current state of affairs which could change at any later date.  The purpose is to have a safety valve because the rake task overrides an relatively important ENV value (and many similar rake spec/ci tasks do so).